src/main/kotlin/no/nav/familie/ks/barnehagelister/rest/BarnehagelisterController.kt

```diff
@@ -23,7 +23,7 @@ import java.util.UUID
     issuer = "maskinporten",
     claimMap = ["scope=nav:familie/v1/kontantstotte/barnehagelister"],
 )
-@RequestMapping("/api/barnehagelister")
+@RequestMapping("/api/kindergartenlists")
```

Comment:
Jeg tror faktisk det er mulig å ha flere ur'ler i requestmapping så slipper man hele oppsettet med separat fil for norske url. Så kan man bare fjerne de når visma har gått over
```suggestion
@RequestMapping("/api/kindergartenlists", "/api/barnehagelister")
```

Reply:
Gøy! Da gjør vi det i stedet!